### PR TITLE
Milestone 4: Hooks

### PR DESCRIPTION
**Pull request by AkashaRojee**
Hello!

I refactored the project with hooks.

Things done:

- Convert class components to functional components
- Implement state hook in Calculator
- Implement memo hook in Keyboard
- Update README
- Update build
- Deploy from feature branch

Please, review the code.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ Calculator
 - Lift state up to Calculator
 - Pass state as props to Display
 
+### 4. Refactor with hooks
+
+- Convert class components to functional components
+- Implement state and memo hooks
+
```
Comment (MahmoudBakr23) [inline README.md:47]: - [x] In the README file, kindly fix the broken issues page link for a completely professional repo 😄 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @AkashaRojee 😃, 

Well done refactoring the code from class-based component to functional component using hooks and ES6 features 💯
Your project is complete! However, there is only one thing to do in order to make the project completely professional and you'll be good to go 🙏 

_Please see the comment down below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AkashaRojee** [comment]
Hi @MahmoudBakr23,

I have now corrected the relative path to the issues page.

Thanks!

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @AkashaRojee 😃, 

Thanks for making the changes 💯 Extra credit for the live demo link 🚀 Good luck with the next and happy coding 🙏 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AkashaRojee** [comment]
Hi @MahmoudBakr23,

Thanks!

**Pull request closed without merging**